src/gscreenshot/__init__.py

```diff
@@ -37,7 +37,24 @@ def __init__(self, screenshooter=None):
 
         self.saved_last_image = False
         self.last_save_file = None
-        self.last_save_directory = os.path.expanduser("~")
+        if os.path.isfile(self.get_save_file()):
+            f = open(self.get_save_file(), "r")
+            self.last_save_directory = f.readline()
```

Comment:
Let's make the cache file a structured data file, like .ini (or something else lightweight). I can see the possibility of adding more things to this (maybe a "quick action" that does what you did the last time you used gscreenshot, for example). If we make it structured data we can build on it.